doc/news/changes/major/20240510Still

```diff
@@ -0,0 +1,3 @@
+Updated: Tetrahedral meshes are now refined quasi-uniformly, preventing degradation with higher refinement levels.
```

Comment:
I would add that also the multigrid infrastructure has been updated to take this into account:
```suggestion
Updated: Tetrahedral meshes are now refined quasi-uniformly, preventing degradation
with higher refinement levels. The changes also include the relevant adjustments in the
geometric multigrid infrastructure.
```
I think it might make sense to unify this with the previous entry https://github.com/dealii/dealii/blob/6495265e5800149683b2ce3f002ba60f64031bff/doc/news/changes/minor/20240320StillFehnKronbichler#L1-L8
I would personally remove that old entry in favor of the new one, not as a way to diminish the contribution but just for clarity to the users not involved in these changes. However, I think we should add another "minor" entry regarding the correction for the restriction matrices you did for `FE_SimplexP`.